tools/peephole-replace.py

```diff
@@ -331,6 +355,26 @@ def process(self, lines):
         0xFA: "ENTITY_PHOTOGRAPHER",
         0xFF: "ENTITY_NONE",
     }),
+    PeepholeRule("""
+        ld   hl, wEntitiesOptions1Table
+        add  hl, de
+        set  @, [hl]
+    """, ENTITY_OPT_BITS),
+    PeepholeRule("""
+        ld   hl, wEntitiesOptions1Table
+        add  hl, bc
+        set  @, [hl]
+    """, ENTITY_OPT_BITS),
+    PeepholeRule("""
+        ld   hl, wEntitiesOptions1Table
+        add  hl, bc
```

Comment:
think you wanted `add hl, de` here, as now you have the same rule twice.

Reply:
oops, indeed - thanks